### PR TITLE
Support slash in GitHub branch name

### PR DESCRIPTION
**Pull request by BlythMeister**
This would previously error

Also updated bootstrapper to version including my fix for group policy restrictions

### CODE REVIEW

**Comment by matthid** [comment]
The only thing is that a test for this function might make sense (we can make it internal and use InternalsVisibleTo if needed)

**Comment by BlythMeister** [comment]
Agreed, is there an already existing test fixture?
Also looks like I broke something as both CI gone red 😞

**Comment by BlythMeister** [comment]
@matthid turns out there is tests (just not for this case) and i broke them :(
Adding my new scenario into that test case and will ensure all tests pass!

**Comment by BlythMeister** [comment]
@matthid updates made, should be ready to go now...

File: tests/Paket.Tests/Lockfile/GeneratorSpecs.fs [hunk 1]
```diff
@@ -177,22 +177,36 @@ let ``should generate lock file with disabled content for packages``() =
...
-github "owner:project2:commit2" "folder/file.fs"
-github "owner:project2:commit3" "folder/file3.fs" githubAuth """ 
+    folder/file3.fs (commit3) githubAuth
+  remote: owner/project3
+    FULLPROJECT (master)"
+
+
+    let config = "github \"owner:project1:master\" \"folder/file.fs\"
+github \"owner:project1:feature/branch\" \"folder/file3.fs\"
+github \"owner/project1:feature:branch\" \"folder/file4.fs\"
```
Comment (matthid) [inline tests/Paket.Tests/Lockfile/GeneratorSpecs.fs]: Should we really allow that?

Comment (BlythMeister) [reply tests/Paket.Tests/Lockfile/GeneratorSpecs.fs]: Don't see why not...you can create a branch called that I believe?

Comment (baronfel) [reply tests/Paket.Tests/Lockfile/GeneratorSpecs.fs]: pretty common with branching strategies like [gitflow](https://datasift.github.io/gitflow/IntroducingGitFlow.html)

Comment (BlythMeister) [reply tests/Paket.Tests/Lockfile/GeneratorSpecs.fs]: @matthid updated to no longer allow this.
Also added a range of other test cases :)

**Comment by forki** [comment]
thx

**Pull request merged**